pytensor/scan/op.py

```diff
@@ -1500,6 +1500,10 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling, impl=None):
         node_input_storage = [storage_map[r] for r in node.inputs]
         node_output_storage = [storage_map[r] for r in node.outputs]
 
+        # HACK: Here to handle Blockwise Scans
```

Comment:
It's not a hack, compute map for inner graphs is useless.

Instead of creating a compute_map make an if/else below where `rval` is returned and return an rval that doesn't try to assign to compute_map if it was None

Reply:
@ricardoV94 is this change alright?